src/components/BTextArea/BTextArea.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div :class="classes">
+    <label v-if="!hideLabel" :for="id">
+      <strong> 
```

Comment:
Why are we using `<strong>` instead of css `font-weight: bold`? I noticed this on Jason's too.

Reply:
Because I copied Jason's.